docs/source/release_notes.rst

```diff
@@ -11,6 +11,7 @@ Release Notes
         * Removed existing nullable type handling across AutoMLSearch to just use new handling :pr:`4085`, :pr:`4043`
         * Handled nullable type incompatibility in ``Decomposer`` :pr:`4105`, :pr:`4043`
         * Changed the default value for ``null_strategy`` in ``InvalidTargetDataCheck`` to ``drop`` :pr:`4131`
+        * Pinned sktime version to 0.17.0 for nullable types support :pr:`4137`
```

Comment:
Very random but I don't see your release note to remove nullable type handling from ARIMA and the ExponentialSmoothingRegressor. Seems it accidentally got removed in https://github.com/alteryx/evalml/pull/4131/files